src/services/expenses.service.js

```diff
@@ -0,0 +1,54 @@
+const { Op } = require('sequelize');
+const { Expense } = require('../models/models').models;
+
+class ExpensesService {
+  getAll({ userId, categories, from, to }) {
+    const where = {};
+
+    if (userId) {
+      where.userId = userId;
+    }
+
+    if (categories) {
+      if (Array.isArray(categories)) {
+        where.category = {
+          [Op.in]: categories,
+        };
+      } else {
+        where.category = categories;
+      }
+    }
+
+    if (from && to) {
+      where.spentAt = {
+        [Op.between]: [from, to],
+      };
+    }
+
+    return Expense.findAll({
+      where,
+    });
+  }
+
+  getById(id) {
+    return Expense.findByPk(id);
+  }
+
+  create(expense) {
+    return Expense.create(expense);
+  }
+
+  async update(id, expense) {
+    await Expense.update(expense, { where: { id } });
```

Comment:
Consider returning the result of the `update` operation to confirm whether the update was successful.